runtime/doc/options.txt

```diff
@@ -5634,6 +5634,26 @@ A jump table for the options with a short description can be found at |Q_op|.
 	generated from a list of items, e.g., the Buffers menu.  Changing this
 	option has no direct effect, the menu must be refreshed first.
 
+						*'messagesopt'* *'mopt'*
+'messagesopt' 'mopt'		string	(default "hit-enter,history:500")
+			global
+
+	Option settings when outputting messages.  It can consist of the following
+	items.  Items must be separated by a comma.
```

Comment:
- Fixed exceeding 80 columns.

```suggestion
	Option settings when outputting messages.  It can consist of the
	following items.  Items must be separated by a comma.
```